lib/oauth/authenticator.js

```diff
@@ -29,13 +29,21 @@ OAuthAuthenticator.prototype.authenticate = function authenticate(req,callback){
   }else{
     throw new Error('authenticate must be called with a request object and callback function');
   }
+
   var authHeader = req && req.headers && req.headers.authorization;
 
   if (authHeader){
     if(authHeader.match(/Bearer/i)){
+      var authenticator = new JwtAuthenticator(self.application);
+
+      if (this.localValidation) {
+        authenticator.withLocalValidation();
+      }
+
       var parts = authHeader.split(' ');
```

Comment:
I didn't mean at the top of this scope, just 4 lines up :)

I find it easier to read because I can see that we have 3 variables; `authenticator`, `parts`, and `token` for this code block, and they are all used together, right? *Setup* and then *execution*...

```javascript
var authenticator = new JwtAuthenticator(self.application);
var parts = authHeader.split(' ');
var token = parts[parts.length-1];

if (this.localValidation) {
  authenticator.withLocalValidation();
}

authenticator.authenticate(token, done);
```

**vs**

```javascript
var authenticator = new JwtAuthenticator(self.application);

if (this.localValidation) {
  authenticator.withLocalValidation();
}

var parts = authHeader.split(' ');
var token = parts[parts.length-1];

authenticator.authenticate(token, done);
```